### PR TITLE
feature (webapi): add custom server header to all responses

### LINKED ISSUE #1049: Return the current version in every response
The Knora version should be returned with every HTTP request, e.g., `Server: webapi/2.1.0`, instead of the `akka-http` version (`Server: akka-http/10.1.3`), which is currently returned.

This information could then be used by a client, to make sure that he is connected to a supported version of the server.

### PR DESCRIPTION
**Pull request by subotic**
# Issues

- resolves #1049

### CODE REVIEW

**Comment by subotic** [comment]
@benjamingeer whenever you have a spare moment. Should be a quick thing.

File: webapi/src/main/scala/org/knora/webapi/http/ServerVersion.scala [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import akka.http.scaladsl.model.HttpResponse
+import akka.http.scaladsl.model.headers.Server
+import akka.http.scaladsl.server.Route
+import org.knora.webapi.BuildInfo
+import akka.http.scaladsl.server.Directives.respondWithHeader
+
+/**
+  * This trait can be used to add the [[Server]] header to an [[HttpResponse]].
+  */
+object ServerVersion {
```
Comment (benjamingeer) [inline webapi/src/main/scala/org/knora/webapi/http/ServerVersion.scala]:
```suggestion
trait ServerVersion {
```

(or change the comment)

Comment (subotic) [reply webapi/src/main/scala/org/knora/webapi/http/ServerVersion.scala]: ah, yes, thanks.

**Comment by subotic** [comment]
Thanks :-)

Unfortunately, not anymore. Today and tomorrow I'm working from home.

**Pull request merged**